pyproject.toml

```diff
@@ -40,6 +40,7 @@ dependencies = [
     "scikit-learn >= 0.22",
     "python-dateutil >= 2.8.1",
     "scipy >= 1.4.0",
+    "importlib-resources >= 5.10.0"
```

Comment:
If I am following correctly, we are adding this because we can't use the new API with Python 3.8. If that is the case, we should probably create an issue to drop this dependency after we have dropped support for Python3.8 as this will no longer be necessary and we can just use `importlib.resources` directly.

Reply:
Agreed, I'm a big fan of removing dependencies

Reply:
Created: https://github.com/alteryx/woodwork/issues/1591